source/trunk/Esf.sln

```diff
@@ -0,0 +1,65 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I think it's good idea to redirect the output of all the projects(the bin folders) to one deployment directory.